README.md

```diff
@@ -106,25 +112,45 @@ Docker image on Docker Hub (see above).
 **Build and install with local PostgreSQL**
 
 ```bash
-# To build the extension
 make
-
-# To install
 make install
 ```
 
-**Update `postgresql.conf`**
+You'll need to [update your PostgreSQL configuration](#update-conf) and
+restart as well.
+
+<a name="update-conf"></a>
```

Comment:
Just a note, when we transfer these over to docs, we need to use `id` instead of `name` as `name` messes with my scrolling function.